config/prow/cluster/build_serviceaccounts.yaml

```diff
@@ -27,4 +27,13 @@ metadata:
   name: k8s-kops-test
   namespace: test-pods
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  annotations:
+    # Kops GCE compute instance service account (nodes and masters)
+    iam.gke.io/gcp-service-account: kops-instance@kubernetes-jenkins-pull.iam.gserviceaccount.com
```

Comment:
I would prefer we not the kubernetes-jenkins-pull project. If we need to use this project them IMO we might as well continue using pr-kubekins.

And ideally this uses a project owned by the community rather than Google (kubernetes-jenkins-pull is a Google owned project). @listx should know what the best practice is for setting this up (he did the k8s-infra-gcr-promoter GCPSA above)

On the other hand, I do have access to pr-kubekins and can run the gcloud command to grant it access to the k8s-kops-instance KSA.

Reply:
I think we're on the same page. 

I would prefer to create a new GSA (in whatever project is appropriate), and a new bucket, if needed. If we can get this GSA set up and give `pr-kubekins` `iam.serviceAccounts.actAs` on it, this should get us back on track.

Reply:
Why would we need to give pr-kubekins anything?

Reply:
I was thinking that we keep the `pr-kubekins` GSA because it's used by all of our jobs and has the permissions required to execute kops and spin up a cluster to a variety of targets.

For GCE, `pr-kubekins` would execute kops. We pass a flag to kops `--gce-service-account=kops-instance@....iam.gserviceaccount.com` to be used for the VMs. So `pr-kubekins` would need serviceAccount user or permission to act as the `kops-instance` service account.